torch_geometric/contrib/flag/flag.py

```diff
@@ -42,41 +102,18 @@ def forward(
         # Code below adapted from:
         #   https://github.com/devnkong/FLAG/blob/main/deep_gcns_torch
         #     /examples/ogb/ogbn_arxiv/main.py#L56
+        #   and
+        #   https://github.com/devnkong/FLAG/blob/main/deep_gcns_torch
+        #     /examples/ogb/attacks.py
         #
         # This example does not assume that the model forward function has a
         #   `perturb` argument
-        # forward = lambda perturb : model(x+perturb, edge_index)[train_idx]
```

Comment:
@tnichols89 I removed a lot of these comments; let me know if you want to keep them in.

Reply:
Certainly do not want to keep them in! Thanks for removing them.